# HG changeset patch
# User Jordi Gutiérrez Hermoso <jordigh@gmail.com>
# Date 1294692221 18000
#      Mon Jan 10 15:43:41 2011 -0500
# Node ID 746609dd54fda92c2c2f1a46e603854cfdd244e3
# Parent  655d5c2d8462869b50feba92b6a6eaea6d9d8f40
Remove Matlab euphemisms in docs and use @file macro for filenames

diff --git a/doc/ChangeLog b/doc/ChangeLog
--- a/doc/ChangeLog
+++ b/doc/ChangeLog
@@ -1,8 +1,14 @@
+2011-01-10  Jordi Gutiérrez Hermoso  <jordigh@gmail.com>
+
+	* interpreter/diagperm.txi: Remove Matlab euphemism.
+	* interpreter/dynamic.txi: Ditto. Also use @file macro for a
+	couple of filenames.
+
 2011-01-10  John W. Eaton  <jwe@octave.org>
 
 	* interpreter/container.txi (Basic Usage and Examples):
 	Document print_struct_array_contents.
 
 2011-01-05  Rik  <octave@nomad.inbox5.com>
 
 	* interpreter/octave.texi: Put statistics detailmenu in correct order.
diff --git a/doc/interpreter/diagperm.txi b/doc/interpreter/diagperm.txi
--- a/doc/interpreter/diagperm.txi
+++ b/doc/interpreter/diagperm.txi
@@ -505,19 +505,19 @@ like:
 
 @item sparse matrix / scalar preserves the sparsity pattern
 
 @item permutation matrix * matrix is equivalent to permuting rows
 @end itemize
 all of these natural mathematical truths would be invalidated by treating
 assumed zeros as numerical ones.
 
-Note that certain competing software does not strictly follow this principle
-and converts assumed zeros to numerical zeros in certain cases, while not doing
-so in other cases.  As of today, there are no intentions to mimic such behavior 
+Note that @sc{Matlab} does not strictly follow this principle and converts
+assumed zeros to numerical zeros in certain cases, while not doing so in
+other cases. As of today, there are no intentions to mimic such behavior
 in Octave.
 
 Examples of effects of assumed zeros vs. numerical zeros:
 
 @example
 Inf * eye (3)
 @result{}
    Inf     0     0
diff --git a/doc/interpreter/dynamic.txi b/doc/interpreter/dynamic.txi
--- a/doc/interpreter/dynamic.txi
+++ b/doc/interpreter/dynamic.txi
@@ -1206,54 +1206,53 @@ oct-file might be
 @c WRITE ME, using Coda section 1.3 as a starting point.
 
 @node Mex-Files
 @section Mex-Files
 @cindex mex-files
 @cindex mex
 
 Octave includes an interface to allow legacy mex-files to be compiled
-and used with Octave.  This interface can also be used to share code
-between Octave and non Octave users.  However, as mex-files expose the
-internal API of an alternative product to Octave, and the internal
-structure of Octave is different to this product, a mex-file can never
-have the same performance in Octave as the equivalent oct-file.  In
-particular to support the manner in which mex-files access the variables
-passed to mex functions, there are a significant number of additional
-copies of memory when calling or returning from a mex function.  For
-this reason, new code should be written using the oct-file interface
-discussed above if possible.
+and used with Octave. This interface can also be used to share code
+between Octave and @sc{Matlab} users. However, as mex-files expose the
+@sc{Matlab}'s internal API, and the internal structure of Octave is
+different, a mex-file can never have the same performance in Octave as
+the equivalent oct-file. In particular to support the manner in which
+mex-files access the variables passed to mex functions, there are a
+significant number of additional copies of memory when calling or
+returning from a mex function. For this reason, new code should be
+written using the oct-file interface discussed above if possible.
 
 @menu
 * Getting Started with Mex-Files::  
 * Working with Matrices and Arrays in Mex-Files::  
 * Character Strings in Mex-Files::  
 * Cell Arrays with Mex-Files::  
 * Structures with Mex-Files::  
 * Sparse Matrices with Mex-Files::  
 * Calling Other Functions in Mex-Files::  
 @c * Application Programming Interface for Mex-Files::  
 @end menu
 
 @node Getting Started with Mex-Files
 @subsection Getting Started with Mex-Files
 
-The basic command to build a mex-file is either @code{mkoctfile --mex} or
-@code{mex}.  The first can either be used from within Octave or from the
-command line.  However, to avoid issues with the installation of other
-products, the use of the command @code{mex} is limited to within Octave.
+The basic command to build a mex-file is either @code{mkoctfile --mex}
+or @code{mex}. The first can either be used from within Octave or from
+the command line. However, to avoid issues with @sc{Matlab}'s own @code{mex}
+command, the use of the command @code{mex} is limited to within Octave.
 
 @DOCSTRING(mex)
 
 @DOCSTRING(mexext)
 
-One important difference between the use of mex with other products and
-with Octave is that the header file "matrix.h" is implicitly included
-through the inclusion of "mex.h".  This is to avoid a conflict with the
-Octave file "Matrix.h" with operating systems and compilers that don't
+One important difference with the use of @code{mex} between @sc{Matlab} and
+Octave is that the header file "matrix.h" is implicitly included through
+the inclusion of "mex.h". This is to avoid a conflict with the Octave
+file "Matrix.h" with operating systems and compilers that don't
 distinguish between filenames in upper and lower case
 
 Consider the short example:
 
 @example
 @group
 @EXAMPLEFILE(firstmexdemo.c)
 @end group
@@ -1665,20 +1664,20 @@ within a mex-file.
 @c @subsection Application Programming Interface for Mex-Files
 @c 
 @c WRITE ME, refer to mex.h and mexproto.h
 
 @node Standalone Programs
 @section Standalone Programs
 
 The libraries Octave itself uses, can be utilized in standalone
-applications.  These applications then have access, for example, to the
-array and matrix classes as well as to all the Octave algorithms.  The
-following C++ program, uses class Matrix from liboctave.a or
-liboctave.so.
+applications. These applications then have access, for example, to the
+array and matrix classes as well as to all the Octave algorithms. The
+following C++ program, uses class Matrix from @file{liboctave.a} or
+@file{liboctave.so}.
 
 @example
 @group
 @EXAMPLEFILE(standalone.cc)
 @end group
 @end example
 
 @noindent
